README.md

```diff
@@ -1,5 +1,51 @@
 # rancher/aks-operator
```

Comment:
We should probably include a description of what this thing does up top. I don't think any have them so lower priority, but long term its good idea.

Reply:
Added